index.js

```diff
@@ -90,10 +118,11 @@ app.post("/signup", (req, res) => {
 	userService
 		.create({ email: req.body.email, password: req.body.inputPassword })
 		.then(result => {
-			res.json(result);
```

Comment:
promise variable result isn't used then

Reply:
removed unused `result`. promise still returns it since we'll probably use it for customizing message later on